scripts/src/main/resources/scripts/setup-docker

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+docker_check=$(docker --version)
+if [[ $docker_check != *"Docker version"* ]];then
+	echo "Installing Docker..."
+	sudo apt-get update -y
```

Comment:
will this sudo require a password to be entered by the user? Do we need to prompt something before to explain what kind of password is actually required here to avoid confusion?

Reply:
Added a hint.